packages/webdriverjs/README.md

```diff
@@ -162,3 +162,13 @@ const axe = new AxeBuilder(driver).setLegacyMode();
 const result = await axe.analyze();
 axe.setLegacyMode(false); // Disables legacy mode
 ```
+
+### WebdriverJS Deque example
```

Comment:
Why mention "Deque" here?

```suggestion
### WebdriverJS Example
```

Reply:
Forgot to remove after using local server as example instead of deque.com